libraries/scriptFile.rb

```diff
@@ -0,0 +1,62 @@
+#
+# Cookbook Name:: jenkins
+# HWRP:: scriptFile
```

Comment:
The filename should be `libraries/script_file.rb` and this comment `# Libraries: script_file_`